trl/trainer/reward_config.py

```diff
@@ -37,3 +37,5 @@ class RewardConfig(TrainingArguments):
     max_length: Optional[int] = None
     """The maximum length of the sequences in the batch. This argument is required if you want to use the default data collator."""
     dataset_num_proc: Optional[int] = None
+    """Coefficient to incentivize the reward model to output mean-zero rewards."""
```

Comment:
```suggestion
    """Coefficient to incentivize the reward model to output mean-zero rewards (proposed by https://huggingface.co/papers/2312.09244, Eq. 2)."""
```

Reply:
following https://github.com/huggingface/trl/issues/1931#issuecomment-2294902065, is it correct?